packages/ui/src/components/MenuV2/components/Item.tsx

```diff
@@ -167,7 +170,7 @@ const Item = forwardRef<HTMLElement, ItemProps>(
               href={href}
               target={target}
               rel={rel}
-              ref={ref as Ref<HTMLAnchorElement>}
+              ref={computedRef as Ref<HTMLAnchorElement>}
```

Comment:
On the last deployment, focus doesn't go inside the menu


https://github.com/user-attachments/assets/e60a2a4f-1912-4989-99c9-ebb728b9f3f5